ml-agents/mlagents/trainers/policy/tf_policy.py

```diff
@@ -144,6 +144,10 @@ def init_load_weights(self):
                 self.assign_ops.append(tf.assign(var, assign_ph))
 
     def load_weights(self, values):
+        if len(self.assign_ops) == 0:
+            logger.warning(
+                "Calling load_weights in tf_policy but assign_ops is empty. Did you forget to call init_load_weights?"
```

Comment:
Is this a warning or should it raise?

Reply:
In the past, we've opted to raise a warning and not exceptions for things like this i.e. adding more than one policy to a ppo trainer.